internal/app/recordtester/recordtester_app.go

```diff
@@ -156,6 +184,19 @@ func (rt *recordTester) Start(fileName string, testDuration, pauseDuration time.
 	mediaURL := fmt.Sprintf("%s/%s/index.m3u8", ingest.Playback, stream.PlaybackID)
 	glog.V(model.SHORT).Infof("RTMP: %s", rtmpURL)
 	glog.V(model.SHORT).Infof("MEDIA: %s", mediaURL)
+
+	if rt.accessControl && (rt.signingKey != "" && rt.publicKey != "") {
```

Comment:
Having `accessControl` and not the signing key should be an explicit fatal error. We should not ignore the setting because another required option was not provided, it is better to just fail the whole test.

```suggestion
	if rt.accessControl {
	  if rt.signingKey == "" || rt.publicKey == "" {
	    msg := "Signing key and public key are required for access control test"
	    glog.Fatal(msg)
	    return 1, errors.New(msg) // (Fatal log above already exits the process but you know just in case, leave it clear here that we end with an error)
	  }
```